src/components/App.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanels from './ButtonPanel';
+
+const App = () => (
+  <div>
```

Comment:
let us use fragments here, the `div` is only there because of the React convention, we can avoid creating extra node by using fragment.`<></>`